gemrb/core/GUI/EventMgr.h

```diff
@@ -74,6 +74,8 @@ class Window;
 #define GEM_MB_SCRLUP           8
 #define GEM_MB_SCRLDOWN         16
 
+#define GEM_MB_ONGOING_ACTION   33
```

Comment:
This looks unused? Also the new Event struct has a repeat count field and possibly other useful attributes that could/should be used instead I think.